### ISSUE TITLE
Probability computation and train disruptions

### ISSUE THREAD

**Issue by roberto-ceraolo** [giacomoorsi/OpenRitardi#24]
A couple of feature ideas that could be interesting/useful:
- Given a journey with changes, can we estimate the probability that the change will be successful?
- How often does a certain train get canceled? Are there some trains that get canceled very often?

**Comment by giacomoorsi** [comment]
Good point. Also to be mentioned that currently we show delays data, but if the train is canceled we don't track this information at all..

**Comment by frasalvi** [comment]
> Given a journey with changes, can we estimate the probability that the change will be successful?

At some point, we could always consider building and incorporating a version of the LSDS robust travel planner...
